packages/reports/addon/templates/components/metric-selector.hbs

```diff
@@ -39,7 +39,7 @@
             @request={{this.request}}
             @onAddParameterizedMetric={{update-report-action "ADD_METRIC_WITH_PARAM"}}
             @onRemoveParameterizedMetric={{update-report-action "REMOVE_METRIC_WITH_PARAM"}}
-            @onToggleParameterizedMetricFilter={{update-report-action "TOGGLE_PARAMETERIZED_METRIC_FILTER"}}
+            @onToggleParameterizedMetricFilter={{update-report-action "TOGGLE_PARAMETERIZED_METRIC_FILTER" this.expandFilters}}
```

Comment:
As mentioned above, we should just queue this `expandFilters` action rather than calling it within the consumer